setup.py

```diff
@@ -1,98 +1,52 @@
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#		http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-# SPDX-License-Identifier: Apache-2.0
-# ============================================================================
-#
-from pathlib    import Path
-from setuptools import setup as setuptools_setup, find_namespace_packages as setuptools_find_namespace_packages
-
-
-gitHubNamespace =       "edaa-org"
-projectName =           "CLITool"
-projectNameWithPrefix = "pyEDAA." + projectName
-version =               "0.1.0"
-
-# Read README for upload to PyPI
-readmeFile = Path("README.md")
-with readmeFile.open("r") as file:
-	long_description = file.read()
-
-# Read requirements file and add them to package dependency list
-requirementsFile = Path("requirements.txt")
-with requirementsFile.open("r") as file:
-	requirements = [line for line in file.readlines()]
-
-# Derive URLs
-sourceCodeURL =     "https://github.com/{namespace}/{projectName}".format(namespace=gitHubNamespace, projectName=projectName)
-documentationURL =  "https://{namespace}.github.io/{projectName}".format(namespace=gitHubNamespace, projectName=projectName)
-
-# Assemble all package information
-setuptools_setup(
-	name=projectNameWithPrefix,
-	version=version,
-
-	author="Patrick Lehmann",
-	author_email="Paebbels@gmail.com",
-	# maintainer="Patrick Lehmann",
-	# maintainer_email="Paebbels@gmail.com",
-  license='Apache 2.0',
-
-	description="EDA Tool CLI Abstraction Layer.",
-	long_description=long_description,
-	long_description_content_type="text/markdown",
-
-	url=sourceCodeURL,
-	project_urls={
-		'Documentation': documentationURL,
-		'Source Code':   sourceCodeURL,
-		'Issue Tracker': sourceCodeURL + "/issues"
-	},
-	# download_url="https://github.com/vhdl/pyVHDLModel/tarball/0.1.0",
-
-	packages=setuptools_find_namespace_packages(exclude=["tests", "tests.*",]),
+"""\
+Package installer for 'An abstraction layer of EDA CLI tools'.
+"""
+from pathlib             import Path
+from pyTooling.Packaging import DescribePythonPackageHostedOnGitHub
+
+gitHubNamespace =        "edaa-org"
+packageName =            "pyEDAA.CLIAbstraction"
+packageDirectory =       packageName.replace(".", "/")
+packageInformationFile = Path(f"{packageDirectory}/__init__.py")
+
+DescribePythonPackageHostedOnGitHub(
+	packageName=packageName,
+	description="An abstraction layer of EDA CLI tools.",
+	gitHubNamespace=gitHubNamespace,
+	sourceFileWithVersion=packageInformationFile,
+	developmentStatus="pre-alpha",
 	classifiers=[
```

Comment:
We can copy https://github.com/edaa-org/pySVModel/blob/main/setup.py#L47

Reply:
Done in 801a1521.